### DISCUSSION TITLE
Proxmox Helper Scripts - ARM64 fork

### DISCUSSION THREAD

**Discussion by asylumexp** [tteck/Proxmox#2788] [Show and tell]
Just wanted to share my fork of the scripts to aim in supporting ARM64, of which I managed to get all but 7 of the containers (haven't worked on the VMs yet) working on ARM64, with four of those having Docker images that could be used instead.

It can be found on [my GitHub under Proxmox](https://github.com/asylumexp/Proxmox).

I aimed to make as minimal of changes as possible in case I stop updating it with the latest commits, that someone could fork my repository and merge the upstream changes without any conflicts.

Just wanted to see @tteck if you were interested in adding a link to the repo in the arch_check function in build.func, so that if someone runs your script on ARM64 they'll see my fork that they could use, which I would appreciate a lot since it would get people to look at my repo, since you said previously that you aren't interested in maintaining ARM64. If you want to have this under your own repo I'm happy to merge and maintain my changes into your own ARM64 seperate repo.

**Comment by srcoggin** [comment]
amazing job. works great, very helpful for my home server. thanks a bunch. glad someone is willing to maintain ARM64

**Comment by tteck** [comment]
While your project is cool, I think it's a little distasteful to take all of the hard work put into the new site and use it as your own, Sure, it's MIT, it's still not right. You have my full permission to use the 'old' version of the site.

**Comment by tteck** [comment]
Even using the database
https://github.com/asylumexp/proxmox-arm64-web/blob/66136f4fbaf1e07ae712b44bc0b9768adea1e9f8/components/Script.tsx#L267

**Comment by asylumexp** [comment]
My bad, I'll take it down and work on something myself at a later date.

**Reply by tteck** [reply]
All website sources are now private. The database will also be moved breaking the links.

**Comment by tteck** [comment]
@asylumexp https://github.com/tteck/Proxmox/commit/574226d993cf649ac5f529068bfd1dc0cda1161f

**Reply by asylumexp** [reply]
Thank you!
